python/pyiceberg/io/__init__.py

```diff
@@ -50,6 +50,10 @@
 S3_SECRET_ACCESS_KEY = "s3.secret-access-key"
 S3_SESSION_TOKEN = "s3.session-token"
 S3_REGION = "s3.region"
+HDFS_HOST = "hdfs.host"
+HDFS_PORT = "hdfs.port"
+HDFS_USER = "hdfs.user"
+HDFS_KERB_TICKET = "hdfs.kerb_ticket"
```

Comment:
For the PyIceberg configuration, we don't have to adhere to the PyArrow arguments. Because we have this flexibility, I would suggest extending `kerb` to `kerberos.`
```suggestion
HDFS_KERB_TICKET = "hdfs.kerberos_ticket"
```